static/app/views/organizationStats/teamInsights/overview.tsx

```diff
@@ -301,6 +304,16 @@ function TeamInsightsOverview({location, router}: Props) {
                 location={location}
               />
             </DescriptionCard>
+            {isInsightsV2 && (
```

Comment:
We can use `<Feature>` component here, better for when organization is ever undefined